src/domain/customers/tableTools/CustomerMessageForm.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { Button, TextArea, TextInput } from 'hds-react';
+import { TFunction } from 'i18next';
+import * as Yup from 'yup';
+import { ObjectSchema } from 'yup';
+import { useTranslation } from 'react-i18next';
+
+import { MessageFormValues, MessageTemplate } from '../types';
+import Text from '../../../common/text/Text';
+import styles from './customerMessageForm.module.scss';
+import Select from '../../../common/select/Select';
+
+type Props = {
+  closeModal: () => void;
+  handleSendMessage?: (message: MessageFormValues) => void;
+};
+
+const getMessageFormValidationSchema = (t: TFunction): ObjectSchema => {
+  return Yup.object().shape({
+    subject: Yup.string().required(t('forms.common.errors.required')),
+    message: Yup.string().required(t('forms.common.errors.required')),
+  });
+};
+
+const mockTemplates: MessageTemplate[] = [
+  {
+    id: '1',
+    name: 'Tiedote',
+  },
+  {
+    id: '2',
+    name: 'Viesti',
+  },
+];
+
+const initialValues: MessageFormValues = {
+  templateId: '1',
+  subject: '',
+  message: '',
+};
+
+export const CustomerMessageForm: React.FC<Props> = ({ closeModal, handleSendMessage }) => {
+  const { t } = useTranslation();
+
+  const templateOptions = mockTemplates.map(({ id, name }) => {
+    return {
+      value: id,
+      label: name,
+    };
+  });
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      onSubmit={(values) => handleSendMessage?.(values)}
+      validateOnBlur={false}
+      validateOnChange={false}
+      validationSchema={getMessageFormValidationSchema(t)}
+    >
+      {({ values, errors, handleChange, handleSubmit }) => {
+        return (
+          <form onSubmit={handleSubmit} className={styles.form}>
+            <div className={styles.heading}>
+              <Text as="h4">{t('customers.message.header')}</Text>
```

Comment:
This className can be applied directly to `Text` component, no need for this outer `div`.

Though I haven't found this className defined in `customerMessageForm.module.scss`.

Reply:
Good catch, the style doesn't exists :D will remove.